test/Test/Database/LSMTree/StateMachine.hs

```diff
@@ -1958,8 +1958,8 @@ arbitraryActionWithVars _ label ctx (ModelState st _stats) =
      ++ [ (2,  fmap Some $ (Action <$> genErrors <*>) $ do
             -- Generate at least a 2-way union, and at most a 3-way union.
             --
-            -- Tests for 0-way and 1-way unions are included in the UnitTests
-            -- module. n-way unions for n>3 lead to large unions, which are less
+            -- Tests for 1-way unions are included in the UnitTests module.
```

Comment:
The point being that there are no 0-way tests (the type forbids it), so it's not actually covered in the unit tests.